Tasks/NuGetInstaller/nugetinstaller.ts

```diff
@@ -73,7 +73,7 @@ async function main(): Promise<void> {
         // locateNuGetExe() will strip them and check for existence there.
         let nuGetPath = tl.getPathInput("nuGetPath", false, false);
         let userNuGetProvided = false;
-        if(tl.filePathSupplied("nuGetPath")){
+        if(nuGetPath !== null && tl.filePathSupplied("nuGetPath")){
```

Comment:
please explain.  What does this have to do with v1 agent?  The lib is carried with the task and that implements tl.filePathSupplied